alpa/device_mesh.py

```diff
@@ -3,12 +3,16 @@
 from collections import defaultdict, namedtuple
 from collections.abc import Iterable
 from functools import partial
+from importlib.metadata import metadata
 from itertools import chain
 import logging
 from operator import attrgetter
 import os
 import time
 from typing import Any, List, Union, Sequence, Tuple, Optional
+import pathlib
+import re
+import tensorstore as ts
```

Comment:
Can you add tensorstore to the dependency list [here](https://github.com/alpa-projects/alpa/blob/main/docs/install/from_source.rst)?